bpi.syndicate.inc

```diff
@@ -543,3 +688,217 @@ function bpi_syndicate_action($bpi_id) {
     drupal_goto('admin/bpi');
   }
 }
+
+/**
+ * Build facets form fields.
+ *
+ * @param array $form
+ *   The form.
+ */
+function bpi_build_search_facets(array &$form) {
+  $facets = bpi_get_facets();
+
+  if ($facets) {
+    $filter_form = array(
+      '#type' => 'fieldset',
+      '#title' => t('Filters'),
+      '#collapsible' => TRUE,
+      '#collapsed' => !_bpi_is_filter_applied(),
+      '#attributes' => array(
+        'class' => array(
+          'bpi-filters',
+        ),
+      ),
+    );
+
+    $facets_form = array(
+      '#type' => 'container',
+      '#tree' => TRUE,
+      '#attributes' => array(
+        'class' => array(
+          'bpi-facets',
+        ),
+      ),
+    );
+
+    $use_chosen = TRUE;
+
+    if ($use_chosen) {
+      $form['#attached']['css'][] = drupal_get_path('module', 'bpi') . '/js/chosen/chosen.min.css';
```

Comment:
Ahh, chosen is a third-party library. These should not be part of the module and you should use libraries to load and include it in the make file for the module.